src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -934,7 +934,11 @@ public boolean verifyVisibilityMessageIsDisplayed(int questionNumber) {
     public boolean verifyVisibilityOptionsIsDisplayed(int questionNumber) {
         return getVisibilityOptions(questionNumber).isDisplayed();
     }
-    
+
+    public WebElement getVisibilityOptionTableRow(int questionNumber, int optionRowNumber) {
+        return getVisibilityOptions(questionNumber).findElement(By.xpath("table/tbody/tr[" + optionRowNumber + "]"));
```

Comment:
@Sumei1009, I am able to select and get the text content and call isDisplayed() on the WebElement retrieved using the css selector you specified above in the other comment. Using the following xpath works too, 
<code>.findElement(By.xpath("(table/tbody/tr|table/tbody/hide)[" + optionRowNumber + "]"))</code>. I am still not convinced that selenium cannot work correctly with that row mentioned. Could you try again? If it does not work, then push your current code where you tried to select  that element. 

That check is the most important part of the test, since it looks like currently this test will pass on master.